### ISSUE TITLE
It is recommended that pm2 list add display custom fields, such as Port, to app description

### ISSUE THREAD

**Issue by zhupengfeivip** [Unitech/pm2#4335]
Can you set some custom fields at pm2 start, such as port, function description, etc., and then display the custom fields in the pm2 list command result

**Comment by Unitech** [comment]
Good idea, feel free to PR something about making the table customizable:
https://github.com/Unitech/pm2/blob/development/lib/API/CliUx.js#L238

**Comment by Unitech** [comment]
<> https://github.com/Unitech/pm2/issues/4329

**Comment by szkrd** [comment]
Hmm, to me even a `PM2_FORCE_CONDENSED_MODE` env var would do, but yes, a comma separated list of columns would be really nice :thinking:

**Comment by jeff-pal** [comment]
Had anyone worked on that feature since then?

**Comment by jeff-pal** [comment]
> Possibility to customize which column to display in "pm2 list" #4631

That link is not available anymore. How can I contribute to implement this feature?

**Comment by jeff-pal** [comment]
> Possibility to customize which column to display in "pm2 list" #4631

@Unitech that link is not available anymore. Had anyone worked on that feature since then? How can I contribute to implement this feature?

**Comment by millette** [comment]
I'd like to customize `pm list` output as well, it's now wider then my terminal which makes it ugly.

I tried to use `cut` but that leaves weird characters, even if I disable colors.

I am currently using `bat --wrap=never` to fix the display, but it's not pretty.
